src/Dynamics/CommentTags.php

```diff
@@ -1,20 +1,25 @@
 <?php
 
-namespace LaravelEnso\Comments\DynamicRelations;
+namespace LaravelEnso\Comments\Dynamics;
 
 use Closure;
 use LaravelEnso\Comments\Models\Comment;
 use LaravelEnso\DynamicMethods\Contracts\Method;
+use LaravelEnso\Users\Models\User;
 
 class CommentTags implements Method
```

Comment:
here you should imlement Relations, not Method and this for all relations

Reply:
Will apply to all other packages.